apps/web/src/lib/components/ui/library/crew-library.svelte

```diff
@@ -383,3 +302,5 @@
 		</Tabs.Content>
 	</Tabs.Root>
 </div>
+
+<CrewLibraryDetails {displayedAgent} {showDetails} on:close={handleClose} />

```

Comment:
Listening the a custom event is not necessary here. If `showDetails` controls the crew library dialog, simply setting `showDetails = false` will close it

Reply:
no i am sending a dispatch